submissions/tosinIshola/HTML/Ecommerce Website.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
@gracioust I think you should remove this file. You created another version already